packages/bvaughn-architecture-demo/components/console/filters/EventType.tsx

```diff
@@ -3,6 +3,8 @@ import { ConsoleFiltersContext } from "@bvaughn/src/contexts/ConsoleFiltersConte
 import { Event } from "@bvaughn/src/suspense/EventsCache";
 import { useContext } from "react";
 
+import Checkbox from "../../../../../src/ui/components/shared/Forms/Checkbox";
```

Comment:
I was struggling getting alias paths working, but I need to move the `Checkbox` component up into the `packages` directory anyways which should clean this up.